heuristics/sniffer.go

```diff
@@ -101,10 +101,12 @@ func isLineTag(line string) bool {
 
 // isLineYamlRestrictive determine if the line is yaml(ish). It parses the line as
 // yaml and returns true only if the following criteria is met:
-// - It is a yaml map
-// - The map key has no spaces
-// - The map key starts with a lowercase letter
-// - If the map value contains spaces, the value must be quoted
+//   - It is a yaml map
+//   - The map key has no spaces
+//   - The map key starts with a lowercase letter
+//   - If the map value contains spaces, the value must be quoted
+//   - If the key is ref, and the value is a URL then skip it (its likely a
+//     comment referencing something)
```

Comment:
nitpick (non-blocking): spelling

```suggestion
//   - If the key is ref and the value is a URL then skip it (it's likely a
//     comment referencing something)
```